PipeableTests/PipeablePage_GotoTests.swift

```diff
@@ -33,7 +33,7 @@ final class PipeablePageGotoTests: PipeableXCTestCase {
     func testGotoWithTimeoutFail() async throws {
         let page = PipeablePage(webView)
         do {
-            try await page.goto("http://localhost:3000/goto/timeout/5", timeout: 1_000)
+            try await page.goto("http://localhost:3000/goto/timeout/5", waitUntil: .domcontentloaded, timeout: 1000)
```

Comment:
Consider extracting the local server url in a variable - `page.goto("\(serverUrl)/goto/timeout/5", ...`.
This is a small refactoring and can be done in a separate PR.

Reply:
I can bundle it here.